docs/api/api/api.txt

```diff
@@ -1433,14 +1433,16 @@ XmlResult: buildinfo
 
 POST /trigger/runservice
 
-  This call needs a token created via "createtoken" command to identify user and
-  package which shall be updated via a source service. The token must be delivered
-  as HTTP header:
+  This call needs a token to identify the user and package which shall be updated
+  via a source service. See the cmd=create in the /person/<userid>/token route.
+
+  The token itself must be delivered as a HTTP header:
     Authorization: Token <TOKEN_STRING>
 
 Parameters
-  project: To be used together with "package" to identify the object.
-  package:
+  project: To be used together with "package" to identify the object. Must not be used
```

Comment:
```suggestion
  project: Must be used in conjunction with the "package" parameter to identify the service to trigger. Must not be used
```

Reply:
hm, it violates our style a bit. One can not check easily the existing parameters here anymore in the list when the "package" parameter is only inside of the text

Reply:
But the `package:` parameter is still present in a comparable fashion to how it used to be there. I'm just wondering what the issue here is?